src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -107,6 +111,131 @@ typealias Gold_Data_Ref_Spec = GoldInternal1_Spec
 typealias Gold_Alias = GoldInternal1
 typealias Gold_Alias_Spec = GoldInternal1_Spec
 
+class Gold_QCollection() : Entity {
+
+    override var internalId = ""
+
+    var name = ""
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+    var age = 0.0
+        get() = field
```

Comment:
same

Reply:
This is code generated. Resolving these as duplicates (there's actually only one change to make).